CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ IF(NOT MSVC)
 ENDIF()
 
 IF(Kokkos_ENABLE_TESTS AND NOT KOKKOS_HAS_TRILINOS)
-  find_package(GTest)
+    find_package(GTest QUIET)
```

Comment:
Is there a reason for the different indentation?

Reply:
This is now aligned with the previous `IF` (line 165).
I can see that most of the time we are using only 2 spaces, so I agree that this change was not necessary.